src/Microsoft.ML.OnnxTransformer/OnnxTypeParser.cs

```diff
@@ -177,10 +177,12 @@ public static int GetDimValue(OnnxCSharpToProtoWrapper.TensorShapeProto.Types.Di
                     break;
                 case OnnxCSharpToProtoWrapper.TensorShapeProto.Types.Dimension.ValueOneofCase.DimParam:
                     // Variable-length dimension is translated to 0.
-                    value = 0;
+                    break;
+                case 0:
```

Comment:
The switch statement is on an enum. Can you please change the case 0: to use ValueOneCase.None?